### PR TITLE
WIP Definitions for audit

### PR DESCRIPTION
**Pull request by jim68000**
These are the definitions for audit and evaluation - not I am not certain about all the values in evaluation - this is a straight copy from screen and the source of some of these values is unknown.

### CODE REVIEW

File: generator/specs/paths/investment.yml [hunk 1]
```diff
@@ -276,6 +276,71 @@
...
+      - in: body
+        name: data
+        description: evaluation booleans
+        required: true
+        schema:
+          $ref: '#/definitions/InvestmentEvaluationConfirmations'
+      responses:
+        201:
+          description: status updated
+  /v3/get/{investmentId}/audit:
```
Comment (reupen) [inline generator/specs/paths/investment.yml]: `get` was probably meant to be `investment`?

File: generator/specs/paths/investment.yml [hunk 2]
```diff
@@ -276,6 +276,71 @@
           description: item updated
         400:
           description: invalid input, object invalid
+  /v3/{investmentId}/evaluation:
```
Comment (reupen) [inline generator/specs/paths/investment.yml]: Missing `/investment` in the URL path (probably should be `/v3/investment/{investmentId}/evaluation``

**Comment by jim68000** [comment]
I've removed evaluation for now as it needs more work

File: generator/specs/definitions/investment_defs.yml [hunk 3]
```diff
@@ -185,6 +185,27 @@
...
+    type: object
+    description: A single audit log entry
+    properties:
+      id:
+        type: string
+        format: uuid
+        example: 'd290f1ee-6c54-4b01-90e6-d701748f0851'
+      user:
+        $ref: '#/definitions/Advisor'
+      items_changed_count:
```
Comment (canni) [inline generator/specs/definitions/investment_defs.yml]: Hmm... And where are the actual changes stored?

**Pull request closed without merging**